source/data-api/custom-endpoints.txt

```diff
@@ -725,14 +723,32 @@ data format specified in the endpoint configuration.
 Authenticate the Request
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-An endpoint function can require requests to include valid credentials
-for an application user in the request headers. You can authenticate
-requests for users that log in with any of the following enabled
-authentication providers:
+If an endpoint is configured to use Application Authentication then you
+must include a valid user access token or login credentials with every
+request.
+
+In general, prefer to use an access token over login credentials when
+possible. The token lets you run multiple requests without
+re-authenticating the user. It also lets you send requests from a web
+browser or other environment that enforces :wikipedia:`CORS
+<Cross-origin_resource_sharing>`.
+
+To use an access token, authenticate the user through an authentication
+provider and then include the access token in the request's
```

Comment:
i rec adjusting phrasing to something like this to make clear the role of app services as an intermediary here.

```suggestion
To use an access token, authenticate the user into an App Services App through an authentication provider.
Then include the access token returned by App Services Authentication in the request's
```

Reply:
(prob should adjust the janky line breaks if going w this phrasing)